build.gradle

```diff
@@ -7,8 +7,8 @@ buildscript {
 
       'kotlin': '1.2.21',
 
-      'androidPlugin': '3.1.0-beta2',
-      'androidTools': '26.1.0-beta2',
+      'androidPlugin': '3.1.0-beta4',
+      'androidTools': '26.1.0-beta4',
```

Comment:
You're just going to have to send another PR tomorrow...

Reply:
Oh, there's a release train?  😄

Reply:
I think so. It's been on Friday for the last few weeks.